contracts/src/models/buildings.cairo

```diff
@@ -260,8 +260,6 @@ impl BuildingProductionCustomImpl of BuildingProductionCustomTrait {
             // take back bonuses given to surrounding buildings if this building is a bonus supplier
             self.update_bonuses_supplied(ref world, delete: true);
         }
-
-        world.write_model(@self);
     }
```

Comment:
can you confirm it's ok to remove this?

Reply:
Yes. This is what was causing issues when trying to destroy a building.
The model is already being written in the calling contract after this internal call, and pausing buildings still works as intended.